readthedocs/core/symlink.py

```diff
@@ -243,7 +241,11 @@ def symlink_single_version(self):
         Link from $WEB_ROOT/<project> ->
                   HOME/user_builds/<project>/rtd-builds/latest/
         """
-        default_version = self.project.get_default_version()
+        default_version = self.get_default_version()
+        if default_version is None:
+            self._log("Default version wrong privacy level.")
```

Comment:
It isn't an error, as it will be triggered in normal operation. I guess it probably isn't worth logging, or setting it to debug more than anything.